package.json

```diff
@@ -6,24 +6,24 @@
-    "deploy-gke-release": "npm run generate-release && graph deploy --ipfs https://subgraph-qa-lum.startbahn.jp/ipfs/ --node https://subgraph-qa-lum.startbahn.jp/admin/ startbahn/startrail-mumbai-release subgraph.mumbai-release.yaml",
-    "deploy-gke-master": "npm run generate-master && graph deploy --ipfs https://subgraph.startrail.io/ipfs/ --node https://subgraph.startrail.io/admin/ startbahn/startrail subgraph.mainnet.yaml",
-    "deploy-gke-polygon": "npm run generate-polygon && graph deploy --ipfs https://subgraph-lum.startrail.io/ipfs/ --node https://subgraph-lum.startrail.io/admin/ startbahn/startrail-polygon subgraph.polygon.yaml",
+    "deploy-gke-develop": "npm run generate-develop && graph deploy --ipfs https://subgraph.startbahn.jp/ipfs/ --node https://subgraph.startbahn.jp/admin/ startbahn/startrail-mumbai-develop subgraph.mumbai-develop.yaml",
+    "deploy-gke-release": "npm run generate-release && graph deploy --ipfs https://subgraph.startbahn.jp/ipfs/ --node https://subgraph.startbahn.jp/admin/ startbahn/startrail-mumbai-release subgraph.mumbai-release.yaml",
+    "deploy-gke-master": "npm run generate-master && graph deploy --ipfs https://subgraph.startrail.io/ipfs/ --node https://subgraph.startrail.io/admin/ startbahn/startrail-polygon subgraph.polygon.yaml",
+    "deploy-gke-polygon": "npm run generate-polygon && graph deploy --ipfs https://subgraph.startrail.io/ipfs/ --node https://subgraph.startrail.io/admin/ startbahn/startrail-polygon subgraph.polygon.yaml",
     "deploy-develop": "npm run generate-develop && graph deploy --debug --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ startbahn/startrail-mumbai-develop --access-token $TOKEN subgraph.mumbai-develop.yaml",
     "deploy-release": "npm run generate-release && graph deploy --debug --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ startbahn/startrail-mumbai-release --access-token $TOKEN subgraph.mumbai-release.yaml",
     "deploy-master": "npm run generate-master && graph deploy --debug --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ startbahn/startrail --access-token $TOKEN subgraph.mainnet.yaml",
     "deploy-polygon": "npm run generate-polygon && graph deploy --debug --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ startbahn/startrail-polygon --access-token $TOKEN subgraph.polygon.yaml",
     "generate-local": "node bin/generate-subgraph-yaml local",
     "generate-develop": "node bin/generate-subgraph-yaml mumbai-develop",
     "generate-release": "node bin/generate-subgraph-yaml mumbai-release",
-    "generate-master": "node bin/generate-subgraph-yaml mainnet",
+    "generate-master": "node bin/generate-subgraph-yaml polygon",
```

Comment:
master we can delete no? so long as we have separate polygon targets ... same with deploy targets above.

Reply:
so don't think we need an alias for 'polygon' and 'master' ..?

Reply:
I think only using plolygon is fine from now on. Let me sync it.